### ISSUE TITLE
Add legacy banner on top of v1 book

### ISSUE THREAD

**Issue by dmathieu** [kubernetes-sigs/kubebuilder#1963]
The [book-v1](https://book-v1.book.kubebuilder.io/) book is about v1 of the project, which is legacy and shouldn't be used for new projects.
However, it can be relatively easy to end up there, and not see you're reading old documentation instead of current one.

My suggestion is to add a warning banner at the top of every page of the v1 book so people visiting the site are warned they are not on the most current version.

/kind documentation

**Comment by DirectXMan12** [comment]
/help

Unfortunately, book-v1 was still using old-school gitbook (vs mdbook, which is what we use now), so implementing this will require figuring out how to override the template for gitbook.

**Comment by dmathieu** [comment]
Yeah, that seems tricky. I tried setting up gitbook locally, but the npm appears to be broken. And all their documentations are now removed in favor of their new product.

**Comment by camilamacedo86** [comment]
Hi @dmathieu, 

You do not need to setup locally. See that when you do the pr with the change you can preview that. More info:  https://github.com/kubernetes-sigs/kubebuilder/blob/master/CONTRIBUTING.md#how-to-preview-the-changes-performed-in-the-docs

**Comment by hickeyma** [comment]
@rashmigottipati Did you get a chance to work on this? If not, I can give it a go.

**Comment by rashmigottipati** [comment]
@hickeyma I haven't had a chance to work on this, yes absolutely, please feel free to work on this. :)

**Comment by hickeyma** [comment]
ok, thanks @rashmigottipati

**Comment by hickeyma** [comment]
PR #2289 fixes this. This can be closed.

**Comment by hickeyma** [comment]
@rashmigottipati @camilamacedo86 Do you mins closing this issue as it is resolved by PR #2289.

**Comment by hickeyma** [comment]
Thanks @camilamacedo86
